F/Format/Compat.toml

```diff
@@ -7,8 +7,11 @@ Compat = "0.62-2.1"
 ["0.7"]
 julia = "0.6-1.3"
 
-["0.7.1-0.8"]
+["0.7.1-1.0"]
 Compat = "1.0.1-2.1"
 
 ["0.8"]
 julia = "1.0-1.3"
+
+["1.0"]
+julia = "1.0-1.1"
```

Comment:
This is a bug https://github.com/JuliaComputing/Registrator.jl/issues/103 so please retrigger the bot when the linked issue is fixed.